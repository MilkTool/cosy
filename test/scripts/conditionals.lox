mut x = true
if (x) print("yes") else print("no")
x = false
if (x) print("yes") else print("no")

print "hi" or 2 // "hi"
print nil or "hello" // "hello"

print "hi" and 2 // 2
print nil and "hello" // nil

var i = 0
for i < 10 {
    print "while #" + i
    i = i + 1
}

for j in 0..10 { print "for #" + j }
