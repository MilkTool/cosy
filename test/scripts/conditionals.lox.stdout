yes
no
hi
hello
2
nil
while #0
while #1
while #2
while #3
while #4
while #5
while #6
while #7
while #8
while #9
for #0
for #1
for #2
for #3
for #4
for #5
for #6
for #7
for #8
for #9
